(DOCSP-32272): Atlas Top 250: Add Atlas to intro, provide Atlas-focused connection string (#806)
mongodb/docs-node: source/fundamentals/connection/connect.txt

   :depth: 2
   :class: singlecol

> This guide shows you how to connect to a MongoDB Atlas instance, 

Reviewer: Suggestion:

I think the terminology Atlas uses is "cluster" (see https://www.mongodb.com/docs/atlas/manage-clusters/)

```suggestion
This guide shows you how to connect to a MongoDB Atlas cluster, 
```

Reviewer 2: > Suggestion:
> 
> I think the terminology Atlas uses is "cluster" (see https://www.mongodb.com/docs/atlas/manage-clusters/)

The term "cluster" actually doesn't include serverless instances, which can also connect via Node driver. If we want to use a term other than "instance," we could use "database deployment" or "deployment" as those are usually used in the Atlas docs to encompass both clusters and serverless

Author: I'll go with "deployment" here to encompass both cases. Thank you both for the suggestions/info!

> This guide shows you how to connect to a MongoDB Atlas instance, 

Reviewer 2: Suggestion: link to Atlas on the first mention on this page
```suggestion
This guide shows you how to connect to a `MongoDB Atlas 
<https://www.mongodb.com/docs/atlas>`__ instance, 
```

> another MongoDB instance, or a replica set using the Node.js driver.

Reviewer: Suggestion:
I think it could be better to use "a MongoDB instance" given that "another" might not make sense in the context of the prior suggestion, which changes "instance" to "cluster".

connect to MongoDB and how it should behave while connected. The following
example shows each part of the connection URI:

.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png
   :alt: Each part of the connection string

> In this example, we connect to an Atlas MongoDB deployment that has a 

Reviewer: Suggestion:
I think "Atlas cluster" would be most recognizable given the prior suggestions.

Author: Given the info that Sarah provided above, I think "deployment" covers both clusters and serverless, so I'll stick with that here unless there's a strong preference for something else.

> DNS SRV record. For more details, refer to the 

Reviewer 2: Change to pass Vale rules
```suggestion
DNS SRV record. For more details, see the 
```

:manual:`DNS Seed List Connection Format
</reference/connection-string/#dns-seed-list-connection-format>` 
documentation. This format offers flexibility in deployment and the 
ability to change the servers in rotation without reconfiguring clients.

.. note::

   To learn how to retrieve your connection string in Atlas, see the
   :atlas:`Atlas driver connection guide </driver-connection>`.

Alternately, you could use ``mongodb`` for the protocol, which specifies

> the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.

Reviewer 2: Nit: line wrap
```suggestion
the :manual:`Standard Connection String Format 
</reference/connection-string/#std-label-connections-standard-connection-string-format>`.
```

> Alternately, you could use ``mongodb`` for the protocol, which specifies
> the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.

Reviewer: Suggestion:

I think it could be good to mention that you must use it when you aren't connecting to a DNS SRV address

```suggestion
If you are connecting to an instance or replica set that does not have a DNS SRV address, you must use the ``mongodb`` protocol, which specifies
the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.
```
